crates/network-primitives/src/block.rs

```diff
@@ -1,6 +1,16 @@
 use alloy_primitives::B256;
 use serde::{Deserialize, Serialize};
 
+#[cfg(not(feature = "std"))]
+use alloc::{vec, vec::Vec};
+#[cfg(feature = "std")]
+use std::{vec, vec::Vec};
```

Comment:
should just use `extern crate alloc;` and no cfgs

Reply:
yeah we should do that in more places

Reply:
(see alloy-core)